build_process/package-management.md

```diff
@@ -3,39 +3,39 @@
 ## Overview
 We use [npm][npmCli] to package and distribute shared code.
 
-Anything that can be shared, can be [packaged as a module][packages]. This is not limited to JavaScript, but can include SCSS/CSS, tooling, and other static assets such as images.
+Anything that can be shared can be [packaged as a module][packages]. This is not limited to JavaScript but can include SCSS/CSS, tooling, and other static assets, such as images.
 
 ## Why?
 npm is the canonical solution to manage dependencies and build systems for front-end projects. The benefits of using npm include:
-- discoverability
-- easy access to packages
-- normalized workflows
-- package stability
-- semantic versioning
+- Discoverability
+- Easy access to packages
+- Normalized workflows
+- Package stability
+- Semantic versioning
 
-Even if we choose a different tool to interact with packages such as [yarn][yarn], or to manage versioning and publishing of packages such as [lerna][lerna], they are still built to be compatible with the npm registry and package format. Understanding the packaging strategies established by npm, will help us to be more productive when using secondary tooling.
+Even if we choose a different tool to interact with packages, such as [yarn][yarn], or to manage versioning and publishing of packages, such as [lerna][lerna], they are still built to be compatible with the npm registry and package format. Understanding the packaging strategies established by npm will help us to be more productive when using secondary tooling.
 
 ## Semantic Versioning
-As publishers, we are highly encouraged to read the [semver specification][semver]. It is very succinct, and knowing it is _well_ worth the time it takes to read.
+As publishers, we are highly encouraged to read the [semver specification][semver]. It is very succinct, and it is _well_ worth the time it takes to read.
```

Comment:
When I read this, I'm wondering why we're highly encouraged to read this. Could you maybe describe a little more why this is well worth the time to read?